src/reflow/mod.rs

```diff
@@ -234,14 +239,13 @@ mod tests {
             assert_eq!(dbg!(chunks).len(), 1);
             let chunk = &chunks[0];
 
-            // TODO add tests with indentation and unbreakables or is this sufficiently covered by Gluon tests?
-            let range = 0..CONTENT.len();
+            let range = 0..chunk.as_str().len();
```

Comment:
This is in bytes, but we want chars. :warning:  It will give the idea of functionality but breaks down as soon as you you introduce emoji chars in your tests.
I'd recommend a couple :cactus: :tophat: :thermometer: to make sure that basic mistake is avoided. Could have prevented #107 and a few which I just uncovered in the process of debugging that.